Standards/scs-0101-w1-entropy-implementation-testing.md

```diff
@@ -7,18 +7,26 @@ supplements:
   - scs-0101-v1-entropy.md
 ---
 
-## Implementation
+## Implementation notes
 
 We presume that almost nothing has to be done (or indeed can be done), as
-long as the CPUs and VM images are reasonably recent; only the flavor and
-image attributes have to be set:
+long as the CPUs and VM images are reasonably recent.
```

Comment:
I think this is overly vague for an implementation note. What does "recent" mean here?

I would prefer if we could list the minimum kernel and qemu/libvirt/openstack versions required here, don't you think? At least the kernel version in the guest vm should be explicitly called out imho as there might be reasons for some guests to run older kernels.

Reply:
True. We should indicate what CPUs and kernel versions are required. I don't know about any qemu/libvirt/openstack requirements.

Reply:
65edea5720ccce61b0d37414e114e2848ffb7d33